### DISCUSSION TITLE
Noob question

### DISCUSSION THREAD

**Discussion by hammersxo** [ImageMagick/ImageMagick#2407] [Help]
{
  | use Image::Magick;
  | my $image = Image::Magick->new();
  | $image->BlobToImage(substr($read_data, $pos2, $size));
  | # $image->Scale('640');
  | $image->Write("$upload_dir/$name");
  | }
The above code, in index.cgi , returns an [roadrunner email](https://roadrunneremails.net) error message : " Need to upload Magick.pm to /bin folder...
What does it mean? Where should I get Magick.pm ?

**Comment by fmw42** [comment]
That is the Perl module for ImageMagick. See https://imagemagick.org/www/script/perl-magick.php
